### PR TITLE
feat(Guild): Edit MFA level

### PR DESCRIPTION
**Pull request by conorwastakenwastaken**
[Docs reference](https://discord.com/developers/docs/resources/guild#modify-guild-mfa-level)

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -2641,6 +2641,7 @@ declare namespace Eris {
     ): Promise<Emoji>;
     editGuildIntegration(guildID: string, integrationID: string, options: IntegrationOptions): Promise<void>;
     editGuildMember(guildID: string, memberID: string, options: MemberOptions, reason?: string): Promise<Member>;
+    editGuildMFALevel(guildID: string, level: 0 | 1, reason?: string): Promise<number>;
```
Comment (bsian03) [inline index.d.ts]: `level` should have its own type

**Review by bsian03 — CHANGES_REQUESTED** [review]
lgtm otherwise

File: index.d.ts (hunk 1 above)
Comment (bsian03) [reply index.d.ts]: Constant, not type

Comment (conorwastakenwastaken) [reply index.d.ts]: Whoopsie, fixed

**Review by bsian03 — CHANGES_REQUESTED** [review]
API actually returns `{ level: number }`. Also please specify that the bot needs to be the guild owner for it to work

**Comment by bsian03** [comment]
Tested on 5da4537, no actual code changes since https://github.com/iiFDCT/eris/compare/5da4537...6bcbb02 - Shard and createGuild edits are PR merges.
Will merge once I get to test typings

File: index.d.ts [hunk 2]
```diff
@@ -1142,6 +1142,9 @@ declare namespace Eris {
     expireBehavior?: string;
     expireGracePeriod?: string;
   }
+  interface MFALevelResponse {
+    level: MFALevel
```
Comment (bsian03) [inline index.d.ts]:
```suggestion
    level: MFALevel;
```

**Review by bsian03 — CHANGES_REQUESTED** [review]

**Review by bsian03 — APPROVED** [review]

**Pull request merged**